src/components/Title.js

```diff
@@ -0,0 +1,45 @@
+import { Component } from "react"
+import { connect } from "react-redux"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import changeTextColor from "../state/actions/title"
+
+const Background = styled.div.attrs({
+  className: "flex justify-center bg-light-purple",
+})``
+
+class Title extends Component {
+  constructor() {
+    super()
+    this.changeColorOnClick = this.changeColorOnClick.bind(this)
```

Comment:
@abdhalees Sorry missed this one first time round. I don't think we need to use `constructor` anymore with new ES6 classes. The reasons I knew about for using `constructor` was to initialise state, which can now be done as:

https://github.com/InFact-coop/discover/blob/b7757b92c494b61cfb6934048fff1ef43bb67bf7/src/App.js#L10

To bind `this` to the method, we can now do it like this: 

https://medium.com/quick-code/react-quick-tip-use-class-properties-and-arrow-functions-to-avoid-binding-this-to-methods-29628aca2e25